server/server.js

```diff
@@ -90,5 +90,50 @@ module.exports = function(port, db, githubAuthoriser) {
         });
     });
 
+    // Produce the same ID for any pair of users, regardless of which is the sender
+    function getConversationID(userAId, userBId) {
+        return userAId < userBId ?
+            userAId + "," + userBId :
+            userBId + "," + userAId;
+    }
+
+    app.post("/api/conversations", function(req, res) {
+        var conversationInfo = req.body;
+        var recipientID = conversationInfo.recipient;
+        var senderID = req.session.user;
+        // Find both the sender and the recipient in the db
+        users.findOne({
+            _id: senderID
+        }, function(err, sender) {
```

Comment:
This is ok for now, but I think it might be worth considering a way to use promises instead of callbacks here.